frameworks/digital-outcomes-and-specialists/questions/brief-responses/availability.yml

```diff
@@ -0,0 +1,22 @@
+name: Earliest start date
+question: When is the earliest {% if brief.lot == "digital-outcomes" %}the team can start?{% elif brief.lot == "digital-specialists" %}the specialist can start work?{% else %}you can recruit participants?{% endif %}
+hint: 'eg 31/12/2017'
```

Comment:
https://www.pivotaltracker.com/story/show/137360327
See suggested hint date in this pivotal story